bench/micro.jl

```diff
@@ -32,7 +32,7 @@ const micros = Dict(
     "ascii-1024" => "x" ^ 1024,
     "unicode" => "ສະ​ບາຍ​ດີ​ຊາວ​ໂລກ!",
     "unicode-1024" => "ℜ" ^ 1024,
-    "bool" => true,
+    "Bool" => true,
```

Comment:
I hope you don't mind, but I'm gonna revert this change since it seems unrelated and none of the other type-like names are capitalized (like Integer or Date).